src/module/apps/characterImport/gearImport/BaseGearParser.ts

```diff
@@ -1,6 +1,7 @@
 import { parseDescription, parseTechnology } from "../BaseParserFunctions.js"
 import EquipmentItemData = Shadowrun.EquipmentItemData;
 import {DataDefaults} from "../../../data/DataDefaults";
+import { iconAssign } from '../../iconAssigner/iconAssign';
```

Comment:
You import something you never use?

Reply:
I originally implemented icon assignment in this file but moved it when I added subtypes, I must have missed deleting it but I’ll fix that